tools/congestion-model/src/main.rs

```diff
@@ -125,7 +142,7 @@ fn run_model(
         workload_name,
         strategy_name,
         &model.progress(),
-        &model.gas_throughput(),
+        &(model.gas_throughput() - warmup_gas_usage),
```

Comment:
random thought: 
perhaps average per round would be easier to interpret? 
`(model.gas_throughput() - warmup_gas_usage) / (num_rounds - num_warmup_rounds)`

Reply:
True. I think dividing by number of shards as well makes even more sense, then we can have "gas per chunk".

Reply:
But then we should also change the unit (PGas is not granular enough) so I think it should not be mixed with this commit to keep a clean git history.